src/GCRealTimeMon/Program.cs

```diff
@@ -21,26 +21,32 @@ public class Options
             [Option(shortName: 'n',
                     longName: "processName",
                     Required = false,
-                    HelpText = "The process name for which the GC Monitoring will take place for - the first process is chosen if there are multiple.")]
+                    HelpText = "The process name for which the GC Monitoring will take place for - the first process is chosen if there are multiple. Either this parameter or -p must be passed for the monitoring to begin.")]
```

Comment:
For another PR: remember to change that behavior because you don't want to have monitor an unexpected process.
If more than 1 process with the given name, should exit.
We could also support ps command to list the monitorable processes like what dotnet counters/trace do (see https://github.com/dotnet/diagnostics/blob/7ebf1c10010e1a505914573535b20d8a6f85cc29/src/Tools/Common/Commands/ProcessStatus.cs#L19 for implementation details)